en/SettingsView/MAVLink.md

```diff
@@ -1,9 +1,63 @@
 # MAVLink Settings
 
 The MAVLink settings (**SettingsView > MAVLink**) allow you to configure options and view information specific to MAVLink communications.
+This includes setting the MAVLink system ID for *QGroundControl*, viewing link quality, and managing log uploads (e.g. to [Flight Review](https://logs.px4.io)).
 
-## Link Status
+![MAVLink settings scren](../../assets/settings/mavlink/overview.jpg)
 
-This shows you the status of MAVLink message transfer over the communications link. A high **Loss rate** could lead to protocol errors with things like parameter download or mission upload/download.
+## Ground Station {#ground_station}
 
-![Link Status](../../assets/settings/MAVLinkLinkStatus.jpg)
+This section sets the MAVLink properties and behaviour of *QGroundControl*.
+
+![Ground Station](../../assets/settings/mavlink/ground_station.jpg)
+
+- **MAVLink System ID:** System ID of *QGroundControl* on the MAVLink network (Default: 255).
+  Vehicles are typically allocated IDs from 1. 
+  You may have to specify another ID if there are multiple ground stations or MAVLink applications on the network. 
+- **Emit heartbeat:** Disable emission of regular MAVLink HEARTBEAT message (Default: True).
```

Comment:
This is interesting. In a network with just one GCS and vehicle there isn't any need for the GCS to emit. I suspect though if you have multi GCS networks then perhaps that is no longer true.

Reply:
I think there are lost gcs heartbeat failsafes. Not sure why you would turn this off.

Reply:
OK, I will change to say that generally speaking you should not, and leave it at that unless we can find a good reason.